.yarn/versions/e209f708.yml

```diff
@@ -0,0 +1,34 @@
+releases:
+  "@yarnpkg/cli": patch
+  "@yarnpkg/core": patch
+  "@yarnpkg/plugin-essentials": patch
+
+declined:
+  - "@yarnpkg/plugin-compat"
+  - "@yarnpkg/plugin-constraints"
+  - "@yarnpkg/plugin-dlx"
```

Comment:
I'm not entire sure about this:
I updated `@yarnpkg/plugin-essentials` which would require a re-deploy of `@yarnpkg/cli` & `@yarnpkg/core`

But Idk if the other plugins have to be updated or not too